### ISSUE TITLE
[BUG] [Linter] H026 False positive on `data-id`

### ISSUE THREAD

**Issue by silverwind** [Riverside-Healthcare/djLint#711]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: macOS 15
 - Python Version: 3.11
 - djLint Version: 1.31.1
 - template language: golang

## Issue
H026 is likely only meant to match on `id` and `class` atttributes but matches on `data-id` as well which seems a unintended parser error.

## How To Reproduce
```html
<div data-id=""></div>
```

## Contents of .djlintrc/pyproject.toml [tool.djlint]
```toml
[tool.djlint]
profile="golang"
```

**Comment by christopherpickering** [comment]
Yes thanks for catching. the regex `\b` and `-` combo will chase me forever lol.
